src/components/LogView/index.js

```diff
@@ -398,16 +398,11 @@ class LogView extends React.Component<Props, State> {
       const sortedBookmarks = [...this.props.bookmarks].sort(
         (a, b) => a.lineNumber - b.lineNumber
       );
-      for (let i = 0; i < sortedBookmarks.length; i++) {
-        const { lineNumber } = sortedBookmarks[i];
-        // don't bother scrolling to first line bc it's visible on render
-        if (lineNumber !== 0) {
-          this.setState({ hasScrolledToFirstBookmark: true }, () => {
-            this.props.scrollToLine(lineNumber);
-          });
-          break;
-        }
-      }
+      const { lineNumber } =
+        sortedBookmarks.find(({ lineNumber }) => lineNumber !== 0) || {};
+      this.setState({ hasScrolledToFirstBookmark: true }, () => {
+        this.props.scrollToLine(lineNumber);
+      });
```

Comment:
Should this be a conditional so we don't try to scroll to an undefined linenumber if it is undefined